grpc-gateway/service/createResource_test.go

```diff
@@ -0,0 +1,106 @@
+package service_test
+
+import (
+	"context"
+	"crypto/tls"
+	"testing"
+
+	"github.com/plgd-dev/cloud/grpc-gateway/pb"
+	"github.com/plgd-dev/cloud/resource-aggregate/commands"
+	"github.com/plgd-dev/cloud/test"
+	testCfg "github.com/plgd-dev/cloud/test/config"
+	oauthTest "github.com/plgd-dev/cloud/test/oauth-server/test"
+	"github.com/plgd-dev/go-coap/v2/message"
+	"github.com/plgd-dev/kit/codec/cbor"
+	"github.com/plgd-dev/kit/log"
+	kitNetGrpc "github.com/plgd-dev/kit/net/grpc"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/status"
+)
+
+func TestRequestHandler_CreateResource(t *testing.T) {
+	deviceID := test.MustFindDeviceByName(test.TestDeviceName)
+	type args struct {
+		req pb.CreateResourceRequest
+	}
+	tests := []struct {
+		name            string
+		args            args
+		want            map[string]interface{}
+		wantContentType string
+		wantErr         bool
+		wantErrCode     codes.Code
+	}{
+		{
+			name: "invalid Href",
+			args: args{
+				req: pb.CreateResourceRequest{
+					ResourceId: commands.NewResourceID(deviceID, "/unknown"),
+					Content: &pb.Content{
+						ContentType: message.AppOcfCbor.String(),
+						Data: test.EncodeToCbor(t, map[string]interface{}{
+							"power": 1,
+						}),
+					},
+				},
+			},
+			wantErr:     true,
+			wantErrCode: codes.NotFound,
+		},
+		{
+			name: "/oic/d - PermissionDenied",
```

Comment:
again, missing success path

Reply:
again i need to have merged https://gitlab.iotivity.org/iotivity/iotivity-lite/-/merge_requests/185